### ISSUE TITLE
Docker permission issue

### ISSUE THREAD

**Issue by lsiem** [berndhofer/freqstart#50]
When running the script with the setup routine, the following error occurs:

```
+ Run project: freqstart_nginx.yml
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/images/create?fromImage=amd64%2Fnginx&tag=stable": dial unix /var/run/docker.sock: connect: permission denied
Error in freqstart.sh in function _fsDockerImage_ on line 264
Error in freqstart.sh in function _fsDockerProjectImages_ on line 414
Error in freqstart.sh in function _fsDockerProject_ on line 726

```
This error occurs several times before the nginx setup routine but the script does not interrupt.

**Comment by berndhofer** [comment]
Since I removed sudo from all docker commands (security improvement), your current user has to be in the docker user group. The prerequisites routine should check for that. Maybe the way I validate this has to be improved. But the error should occured earlier too, not just the nginx function. Thank you for the input!

**Comment by berndhofer** [comment]
Switched sudo newgrp to sudo su, because newgrp switched to root instead of the user itself. Now i understand what you meant with "logging into root". Packed everything in a new function. Solved in v1.0.3

**Comment by berndhofer** [comment]
@lsiem can you please test the develop branch. If it's still happening can you look at the file/folder permissions? Thank you!

**Comment by lsiem** [comment]
The script exits after adding the user to the docker group:

```
Installed: docker-ce
User "freqstart_user" added to user group: docker
```

**Comment by berndhofer** [comment]
Ok, at least that works. Since it logsout/in to the same user so that the new grp gets recognized the script stops and I've to output a message/info to restart the script. There is no way around this. Please start again and see if permission errors still occur. Thank you!

Holen Sie sich Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by lsiem** [comment]
I ran into this problem after re-running the script: [#51 ](https://github.com/berndhofer/freqstart/issues/51#issue-1326880815)
Maybe running `su -` followed by `exit` is enough to re-login the user

**Comment by berndhofer** [comment]
Im testing a new approuch with:
`sg ${_group} "newgrp `id -gn`"`

**Comment by berndhofer** [comment]
Stayed with your suggestion with ```sudo su -``` followed by ```exit 0``` incl. a warning message. Implemented in v1.0.3
